metron-analytics/metron-profiler-client/src/main/java/org/apache/metron/profiler/client/stellar/WindowLookback.java

```diff
@@ -94,10 +93,10 @@ public Object apply(List<Object> args, Context context) throws ParseException {
 
   @Override
   public void initialize(Context context) {
-    windowCache = CacheBuilder.newBuilder()
-                              .maximumSize(200)
-                              .expireAfterAccess(10, TimeUnit.MINUTES)
-                              .build();
+    windowCache = Caffeine.newBuilder()
```

Comment:
It looks like you're swapping Google's cache for Caffeine - per our performance findings in enrichments, this would be my preference as well, but strictly speaking should this be part of a classpath and Maven dependency PR?

Reply:
This was done so that `metron-profiler-client` no longer depends on Guava.  Guava was not used here very deeply and removing Guava takes away one huge source of conflicts.

Reply:
I'd like to make the change as small as possible.  I could certainly see if I can get by without removing the Guava dependency.  It will just take some work.

Reply:
Actually, I think what you did is reasonable - it's part of the unit of work. I didn't catch that our caching was actually part of Guava.